CHANGES.md

```diff
@@ -2,7 +2,7 @@
 
 #### next release (8.2.25)
 
-- [The next improvement]
+- Change icon used for display group remove button
```

Comment:
Please don't forget to move this under the latest release heading from `main`.

Reply:
Will do thanks @na9da